### PR TITLE
Add tut to parse markdown files

### PR DESCRIPTION
**Pull request by jamescway**
Adds a /tut directory in project root to store markdown files.
Parses MD files and stores output in target/scala-N.NN/tut.

Refs: #2, #54

### CODE REVIEW

**Comment by jamescway** [comment]
fixing the errors in the markdown

File: build.sbt [hunk 1]
```diff
@@ -77,10 +77,16 @@ lazy val docSettings = site.settings ++ ghpages.settings ++ unidocSettings ++ Se
   unidocProjectFilter in (ScalaUnidoc, unidoc) := inAnyProject
 )
 
+lazy val tutExtraSettings = Seq(
+  tutSourceDirectory := baseDirectory.value / "tut"
```
Comment (trane) [inline build.sbt]: let's follow this conventions set by cats: https://github.com/non/cats/tree/master/docs

`docs/src/{main/tut,site}`

File: build.sbt [hunk 2]
```diff
@@ -77,10 +77,16 @@ lazy val docSettings = site.settings ++ ghpages.settings ++ unidocSettings ++ Se
   unidocProjectFilter in (ScalaUnidoc, unidoc) := inAnyProject
 )
 
+lazy val tutExtraSettings = Seq(
```
Comment (trane) [inline build.sbt]: take a look at this for integrating with the `docSettings` from above: https://github.com/non/cats/blob/master/build.sbt#L55-L81

File: build.sbt (hunk 1 above)
Comment (jamescway) [inline build.sbt]: it seems like when I put the `tutExtraSettings` in `docSettings` or `allSettings` it doesn't get evaluated.  Any idea why?

**Comment by jamescway** [comment]
theres a lot of new boilerplate in the docs, maybe worth trying to silence

**Comment by trane** [comment]
:) nice, it's a great first step.

**Pull request merged**